pyproject.toml

```diff
@@ -94,7 +95,7 @@ force-exclude = '''
     | buck-out
     | build
     | dist
-    | src/zino/mibdumps
+    | src/zino/snmp/mibdumps
```

Comment:
```suggestion
    | src/zino/snmp/mibdumps
    | src/zino/snmp/mibs
```

Reply:
And also ignore that one for ruff

Reply:
That would be useless, both for ruff and black.  The new directory contains MIB modules in ASN format and would never be looked at by Python linter tools.

Reply:
Ah, I wrongly assumed that these would also be `.py` files, my bad